ee/vellum_ee/workflows/display/nodes/vellum/final_output_node.py

```diff
@@ -47,6 +48,14 @@ def serialize(self, display_context: WorkflowDisplayContext, **kwargs: Any) -> J
             "display_data": self.get_display_data().dict(),
             "base": self.get_base().dict(),
             "definition": self.get_definition().dict(),
+            "outputs": [
+                {
+                    "id": str(self._get_output_id()),
+                    "name": "node_input",
```

Comment:
```suggestion
                    "name": "value",
```

addresses the blocking comment above. In fact, I've started using direct references, eg `FinalOutputNode.Outputs.value.name`